apps/election-manager/.eslintrc.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
     'import/resolver': {
       node: {
         extensions: allExtensions,
+        moduleDirectory: ['node_modules', 'src/'],
```

Comment:
Why?

Reply:
Ah yeah I wanted to ask you about this, I was suddenly getting build errors about not being able to find lodash (even though it was installed) midway through working on this branch and this was the only way I could figure out how to fix it and get things building again but I don't actually understand what I'm doing here.